app/views/suggestions/index.erb

```diff
@@ -0,0 +1,50 @@
+<h1>All Suggestions</h1>
+<p><a href="<%="/users/#{@user.id}"%>">Back to profile</a></p>
+<% @sorted_suggestions.each do |suggestion| %>
+  <div id="suggestion">
+    <ul>
+      <h3><a href="<%="/suggestions/#{suggestion.id}"%>"><%=suggestion.title%></a></h3>
+      <li><%=suggestion.description%></li>
+    </ul>
+    <div>
+        <form action ="/suggestions/<%=suggestion.id%>/upvote" method="post" id="upvote">
+          <input type="hidden" name="suggestion_id" value="<%= params[:vote] %>">
```

Comment:
Don't call the params from the views. Pass what you need in the views as instance variables from the controller that renders this view.